### PR TITLE
Metadata deployment

### PR DESCRIPTION
**Pull request by ofersadan85**
Changes:
- Ansible will run the full `deploy.sh` script instead of just `docker compose up`
- `deploy.sh` now includes updating the `*.env` files from gcp metadata
- Added cronjob for `deploy.sh` every 15 minutes (will be set up through ansible)
- Changed `*.env` file permissions to be writable because they need to be updated
- Changed `*.env` filenames to be more general because they will not be used only for api

### CODE REVIEW

File: server/deploy.sh [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
-cd ${HOME}
-docker compose pull
-docker compose up -d --remove-orphans
+gcp_metadata_json_to_env() {
+    # This function is used to get metadata from the GCP metadata server and write it to a file
+    # The value is expected to be in json format and the output will be a .env file
+    # The first parameter is the metadata key to retrieve
+    # The second parameter is the file to write the metadata to
+    METADATA_URL='http://metadata.google.internal/computeMetadata/v1/project/attributes'
+    curl -fsL "${METADATA_URL}/${1}" -H "Metadata-Flavor: Google" | jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]" > "${2}"
```
Comment (ohad24) [inline server/deploy.sh:9]: What is the `${2}` means ?

**Review by ohad24 — APPROVED** [review]
Woke up this morning and doing great job
please merge and update release whenever you ready.

File: server/deploy.sh (hunk 1 above)
Comment (ohad24) [reply server/deploy.sh:9]: The second output ?

Comment (ofersadan85) [reply server/deploy.sh:9]: See line number 7:
> The second parameter is the file to write the metadata to

**Pull request merged**